vendors/nuvoton/boards/numaker_iot_m487_wifi/aws_demos/config_files/FreeRTOSConfig.h

```diff
@@ -143,10 +143,10 @@ void vLoggingPrintf( const char * pcFormat,
                      ... );
 
 /* Map the FreeRTOS printf() to the logging task printf. */
-#define configPRINTF( x )          vLoggingPrintf x
-
+//#define configPRINTF( x )          vLoggingPrintf x
+#define configPRINTF( x )            printf x
 /* Non-format version thread-safe print */
-#define configPRINT( X )                  vLoggingPrint( X )
+//#define configPRINT( X )                  vLoggingPrint( X )
```

Comment:
Please revert the changes to this file

Reply:
@muneebahmed10 , it's done to enable vLogging message after drop the last commit of FreeRTOSConfig.h .